examples/generic_mpcc/mpec_no_s_stationarity.m

```diff
@@ -8,17 +8,20 @@
 
 % set options
 mpccsol_opts = nosnoc.solver.Options();  
-% mpccsol_opts.homotopy_steering_strategy = "Direct";
+mpccsol_opts.homotopy_steering_strategy = "Direct";
 % mpccsol_opts.homotopy_steering_strategy = "ELL_1";
-mpccsol_opts.homotopy_steering_strategy = "ELL_INF"; 
+% mpccsol_opts.homotopy_steering_strategy = "ELL_INF"; 
 mpccsol_opts.calculate_stationarity_type = 1;
 
-mpccsol_opts.complementarity_tol = 1e-12;
+
+mpccsol_opts.complementarity_tol = 1e-20;
```

Comment:
I am not surprised it doesn't work. Ideally we should throw an error if someone selects a complementarity tolerance this small as it is lower than machine precision (about `1e-15`). Though I will check whether there is a structural issue here.

Reply:
Yea the homotopy fails to converge as the best complementarity residual it can get to is `1.58e-17` and never reaches `1e-20 ` which we view as a failure of `mpccsol` to converge. I think this is a willnotfix.